sdk/monitor/monitor-opentelemetry-exporter/samples-dev/httpSample.ts

```diff
@@ -18,21 +18,22 @@
  *
  * @summary demonstrates OpenTelemetry http Instrumentation. It is about how OpenTelemetry will instrument the Node.js native http module.
  */
-import api from "@opentelemetry/api";
+import * as api from "@opentelemetry/api";
```

Comment:
general comment.
As a general practice this can lead to a LOT of code getting included when not needed and as a sample this will get copied...

suggestion: update to just drag in the required components (exports) from the api package.

Reply:
Updated the import here.